### PR TITLE
Added cute beaver and elephant ASCII Art

### PR DESCRIPTION
**Pull request by localleon**
# Added Beaver and Elephant Ascii Art

Added a beaver and elephant to the possible PID Error Arts in errorArt.go 

Addes onto #73 

- [X ] I have read the [contribution guidelines](https://github.com/pesos/grofer/blob/master/CONTRIBUTING.md) and followed it as far as possible. 
- [X ] I have performed a self-review of my own code (if applicable)
- [X ] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [X ] I have run `go fmt` on my code ([reference](https://blog.golang.org/gofmt))
- [X ] My changes generate no new warnings 

This PR seems to be related to #73 , the creator also implements an elephant as ASCI Art. #73 seems to be receiving no new activity. 

Example: 
```

    /   \          /   \
    \_   \        /  __/
    _\   \      /  /__
    \___  \____/   __/
        \_       _/
          | @ @  \_
          |
        _/     /\
        /o)  (o/\ \_
        \_____/ /
          \____/
  Whoooopsssss, invalid PID. 
  Please enter a valid PID.
  
----------------------------------------------------------------------------------------
                         ____
                    ---'-    \
      .-----------/           \
     /           (         ^  |   __
&   (             \        O  /  / .'
'._/(              '-'  (.   (_.' /
     \                    \     ./
     |    |       |    |/ '._.'
     )   @).____\|  @ |
 .  /    /       (    |           Pawoo. Pawoo. Pawoo!
\|, '_:::\  . ..  '_:::\ ..\).    Plz give Elephant a valid PID.
```

### CODE REVIEW

File: src/utils/errorArt.go
Comment (MadhavJivrajani) [inline src/utils/errorArt.go:42]: Hi!
Super adorable! Seems like you missed a function call here!

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
Hi,
If it's not too much to ask for, could you convert the names of all existing functions that print the ascii arti in `errorArt.go` to start with lower case? This will ensure that these functions are not exportable

Ex:
`ErrorDoggo()` -> `errorDoggo()`

File: src/utils/errorArt.go
Comment (MadhavJivrajani) [inline src/utils/errorArt.go:27]: You will probably have to change this to 7 instead of 6 for 6 different function calls, since `rand.Intn(n)` gives a pseudorandom number in the range [0, n) (n not included).

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]

File: src/utils/errorArt.go
Comment (localleon) [reply src/utils/errorArt.go]: Fixed in newest commit.

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
Everything looks good to me!

Could you remove comments before each of the `errorAnimal()` function calls since they are no longer exported?

**Comment by localleon** [comment]
> Everything looks good to me!
> 
> Could you remove comments before each of the `errorAnimal()` function calls since they are no longer exported?

Done.

**Review by MadhavJivrajani — APPROVED** [review]
Awesome! 
The moose and the elephant are amazing xD
Thanks for your contribution :smile: 

Will wait for approval from either @Gituser143 or @metonymic-smokey before merging.

**Review by Gituser143 — APPROVED** [review]

**Pull request merged**
